graph/api/src/main/java/org/jboss/windup/graph/model/ApplicationProjectModel.java

```diff
@@ -3,10 +3,10 @@
 import com.tinkerpop.frames.Property;
 import com.tinkerpop.frames.modules.typedgraph.TypeValue;
 
-@TypeValue(ApplicationModel.TYPE)
-public interface ApplicationModel extends WindupVertexFrame
+@TypeValue(ApplicationProjectModel.TYPE)
+public interface ApplicationProjectModel extends ProjectModel
```

Comment:
This isn't just changing a name, this is changing the meaning, and it seems to me that the new meaning is less useful.

Why not just add the ApplicationModel marker interface to a ProjectModel vertex if you want to be able to differentiate them by type?

Reply:
See above. I want to be able to ask for an app input paths or app projects, separatedly. Just marking them would not allow that. Plus, using it as general marker interface closes the possibility to have Project or File specific methods, relations and properties.

Reply:
With the marker interface, you could have done "new GraphService<ApplicationModel>(event, ApplicationModel.class).findAll()". Now, that is impossible, so I don't see how getting rid of the ApplicationModel marker has helped you.

Reply:
What is that good for? It is never used anywhere.